src/i18n/strings/fi.json

```diff
@@ -3203,5 +3203,6 @@
     "Creating HTML...": "Luodaan HTML:ää...",
     "No virtual room for this room": "Tällä huoneella ei ole virtuaalihuonetta",
     "Switches to this room's virtual room, if it has one": "Vaihtaa tämän huoneen virtuaalihuoneeseen, mikäli huoneella sellainen on",
-    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta"
+    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta",
+    "All": "Kaikki"
```

Comment:
Please don't edit the translation files directly (other than `en_EN.json`). Weblate is the source of truth for these files, so you'll need to add the translation there once this PR is merged.

Reply:
Thanks! Removed the Finnish translation.